src/jit/emitarm64.cpp

```diff
@@ -7991,19 +8042,19 @@ BYTE*               emitter::emitOutputLJ(insGroup  *ig, BYTE *dst, instrDesc *i
     }
     else if (loadLabel)
     {
-        if (fmt == IF_LS_1A)       // LS_1A   XX......iiiiiiii iiiiiiiiiiittttt      Rt simm21
+        if (fmt == IF_LS_1A)       // LS_1A   XX...V..iiiiiiii iiiiiiiiiiittttt      Rt simm21
         {
             // INS_ldr or INS_ldrsw (PC-Relative)
 
             // Is the target a vector register?
             if (isVectorRegister(id->idReg1()))
             { 
-                code &= 0x3FFFFFFF;                                  // clear the size bits
-                code |= insEncodeDatasizeVLS(code, id->idOpSize());  // XX
+                code |= insEncodeDatasizeVLS(code, id->idOpSize());  // XX V
                 code |= insEncodeReg_Vt(id->idReg1());               // ttttt
             }
             else
             {
+                code |= insEncodeDatasizeLS(code, id->idOpSize());   // XX
```

Comment:
I realized It's a bit tricky to extend `insEncodeDatasizeLS` which seems error-prone.
For now, I just use `insEncodeDatasizeVLS` for both vector/integer cases that handles LDR (literal) instead.